cms/djangoapps/contentstore/views/course.py

```diff
@@ -1646,3 +1649,25 @@ def _get_course_creator_status(user):
         course_creator_status = 'granted'
 
     return course_creator_status
+
+
+def _get_library_creator_status(user):
+    """
+    Helper method for returning the library creation status for a particular user,
+    taking into account the values of DISABLE_LIBRARY_CREATION and LIBRARIES_ENABLED.
+    """
+    if user.is_staff:
+        library_creator_status = 'granted'
```

Comment:
Can you define these strings somewhere as constants and/or document what the possible values are?